sheepdog/transactions/upload/__init__.py

```diff
@@ -153,7 +156,7 @@ def unpack_bulk_wrapper(wrapper):
     )
 
 
-def _add_wrapper_to_bulk_transaction(transaction, wrapper, index):
+def _add_wrapper_to_bulk_transaction(transaction, wrapper, index, is_gdc=False):
```

Comment:
The transaction class comes with a copy of the flask config, you can use that instead of adding a new parameter.

https://github.com/uc-cdis/sheepdog/blob/release/versioning/sheepdog/transactions/transaction_base.py#L78

eg
```python
transaction._config.get('is_gdc')
```

Reply:
oh, I see. good catch. Thanks!